client/blocks/app-banner/docs/example.jsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import AppBanner from '../';
+import { PREFERENCE_NAME } from '../utils';
+import { savePreference } from 'state/preferences/actions';
+
+function AppBannerExample( clearPreference ) {
+	return (
+		<div className="docs__design-assets-group">
+			<h2>
+				<a href="/devdocs/blocks/app-banner">App Banner</a>
+			</h2>
+			<AppBanner userAgent="android" />
+			<Button onClick={ clearPreference }>Reset Dismiss Preference</Button>
+		</div>
+	);
+}
+
+const ConnectedAppBannerExample = connect(
+	null,
+	{ clearPreference: savePreference( PREFERENCE_NAME, null ) }
+)( AppBannerExample );
+
+ConnectedAppBannerExample.displayName = 'AppBannerExample';
```

Comment:
We need to do that for examples, or else we mangle the name in wpcalypso. Eg `AppBannerExample` turns into `t` for example.